pkg/utils/apiutil/apiutil.go

```diff
@@ -336,6 +336,22 @@ func ParseKey(name string, input map[string]any) ([]byte, string, error) {
 	return returned, rawKey, nil
 }
 
+// ParseHexKeys decodes hexadecimal keys to bytes if the format is "hex".
+func ParseHexKeys(format string, keys ...*string) error {
```

Comment:
Can add some unit tests for this function

Reply:
added